src/goose.rs

```diff
@@ -2461,17 +2507,18 @@ mod tests {
 
         // Make a POST request to the mock http server and confirm we get a 200 OK response.
         assert_eq!(mock_comment.times_called(), 0);
-        let goose = user.post(COMMENT_PATH, "foo").await;
-        let unwrapped_response = goose.response.unwrap();
-        let status = unwrapped_response.status();
-        assert_eq!(status, 200);
-        let body = unwrapped_response.text().await.unwrap();
-        assert_eq!(body, "foo");
+        if let Ok(goose) = user.post(COMMENT_PATH, "foo").await {
```

Comment:
Just double unwrap() instead IMHO

Reply:
We can't double unwrap the response here as we need both the `request` and the `response`, but fixing this to use `.expect()` instead so we helpfully panic if there's an unexpected error during the test.

Reply:
fixed